src/VCEL.CSharp/Expression/Func/DefaultCSharpFunctions.cs

```diff
@@ -3,116 +3,97 @@
 using System.Linq;
 using VCEL.Core.Expression.Func;
 
-namespace VCEL.CSharp.Expression.Func
+namespace VCEL.CSharp.Expression.Func;
+
+public class DefaultCSharpFunctions : IFunctions<string>
```

Comment:
looks like we did not added these new function support to javascript vcel?

Reply:
Updated with JS function implementations in latest commit.